server/models/Request.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  start: {
+    type: Date,
+    required: true,
+  },
+  end: {
+    type: Date,
+    required: true,
+  },
+  accepted: {
+    type: Boolean,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    default: false,
+  },

```

Comment:
Having two fields to track the status of a booking will often lead to unnecessary complexity and cause more problems down the road. I'd recommend reducing these two to a single field such as `status`. You can maybe throw in an [enum validator](https://mongoosejs.com/docs/schematypes.html#string-validators) there too!

Reply:
Thanks @ugur-akin , I will make it done.